pkg/controller/petset/pet.go

```diff
@@ -103,8 +112,9 @@ func (p *petSyncer) Sync(pet *pcb) error {
 		return p.Update(realPet, pet)
 	}
 	if p.blockingPet != nil {
-		glog.Infof("Create of %v in PetSet %v blocked by unhealthy pet %v", pet.pod.Name, pet.parent.Name, p.blockingPet.pod.Name)
-		return nil
+		message := errUnhealthyPet(fmt.Sprintf("Create of %v in PetSet %v blocked by unhealthy pet %v", pet.pod.Name, pet.parent.Name, p.blockingPet.pod.Name))
+		glog.Info(message)
```

Comment:
The issue here is that all other logs in this file don't provide log level. Maybe there should be an issue for that? I think it's out of the scope of this PR to fix this.